templates/plugin/python/module.conf.erb

```diff
@@ -1,7 +1,7 @@
   Import "<%= @module %>"
 
   <Module "<%= @module %>">
-  <% @config.sort.each do |key,value| -%>
-    <%= key -%> <%= value -%>
-  <% end %>
+  <%- @config.sort.each do |key,value| -%>
+    <%= key %> <%= value %>
+  <%- end -%>
```

Comment:
Just tested with multiple modules. 
It results with config file:

```
# Generated by Puppet
<Plugin "python">
  ModulePath "/usr/lib64/collectd"
  LogTraces false
  Interactive false

  Import "elasticsearch"

  <Module "elasticsearch">
    Cluster test
    Verbose false
    Version 1.0
  </Module>

  Import "rabbitmq"

  <Module "rabbitmq">
    Host "localhost"
    Password "ppp"
    Port "15672"
    PrefixVhost true
    Realm "RabbitMQ Management"
    Username "uuu"
  </Module>

</Plugin>
```

I think newlines between modules looks ok here.